src/nri/config.go

```diff
@@ -27,13 +27,15 @@ type Config struct {
 	ExporterBindPort    string
 	ScrapeInterval      time.Duration
 	HeartBeatPeriod     time.Duration
+	ExtraCollectors     string
 }
 
 type configYml struct {
 	FilterEntity        map[string][]string `yaml:"include_matching_entities"`
 	ExporterBindAddress string              `yaml:"exporter_bind_address"`
 	ExporterBindPort    string              `yaml:"exporter_bind_port"`
 	ScrapeInterval      string              `yaml:"scrape_interval"`
+	ExtraCollectors     string              `yaml:"exporter_extra_collectors"`
```

Comment:
even though it is great to "extend" the list of collectors I fear that it could become misleading 😕 

have you talked about Kai about a possible name?